config/locales/en.yml

```diff
@@ -1,6 +1,7 @@
 ---
 en:
   spree:
+    master: master
```

Comment:
This key does not exist in https://github.com/spree/spree/blob/2-4-stable/core/config/locales/en.yml